pkg/policy/mapstate.go

```diff
@@ -269,7 +269,7 @@ func (e *MapStateEntry) DatapathEqual(o *MapStateEntry) bool {
 		return e == o
 	}
 
-	return e.IsDeny == o.IsDeny && e.ProxyPort == o.ProxyPort
+	return e.IsDeny == o.IsDeny && e.ProxyPort == o.ProxyPort && e.AuthType == o.AuthType
```

Comment:
nit: Please update the function description.

Reply:
@mhofstetter If you want to push a quick fix for this (without rebasing!), I'm happy to then merge without rerunning CI.

Reply:
thanks, fixed the documentation